web/web/api/v1/meter.py

```diff
@@ -102,25 +106,40 @@ def add_meter_info():
     '''Add new meter to database'''
 
     #retrieve input
-    meter_id = request.form.get('meter_id')
-    utility_id = int(request.form.get('utility_id'))
-    service_location_id = request.form.get('service_location_id')
-    feeder = request.form.get('feeder')
-    substation = request.form.get('substation')
-    meter_type = request.form.get('meter_type')
-    is_active = request.form.get("is_active").upper() == "TRUE"
-    is_archived = request.form.get("is_archived").upper() == "TRUE"
+    new_meter = request.get_json()
+
+    meter_id = new_meter['meter_id']
+    utility_id = new_meter['utility_id']
+    service_location_id = new_meter['service_location_id']
+    feeder = new_meter['feeder']
+    substation = new_meter['substation']
+    meter_type = new_meter['meter_type']
+    is_active = new_meter['is_active'].upper() == "TRUE"
```

Comment:
If I don't pass in `is_active` in the body of the request, will this blow up?

Reply:
My assumption is that is_active will always be set to the value False if True is not selected

Reply:
you mean it'll always be set to False on the client? Meaning the web client?
However, if I interact with this endpoint via the cmd line or postman, I won't know about this default. We should handle all of these types of assumptions on the server (backend) layer. We can never guarantee what type of bad data a client may be sending.